build.sbt

```diff
@@ -33,9 +33,12 @@ addCommandAlias("fmt", "all scalafmtSbt scalafmt test:scalafmt")
 addCommandAlias("check", "all scalafmtSbtCheck scalafmtCheck test:scalafmtCheck")
 addCommandAlias(
   "compileJVM",
-  ";coreJVM/test:compile;stacktracerJVM/test:compile;streamsTestsJVM/test:compile;testTestsJVM/test:compile;testTestsJVM/test:compile;testRunnerJVM/test:compile;examplesJVM/test:compile"
```

Comment:
Why not coreJVM here?

Reply:
I was just trying to align with the other commands that included `streamsTestsJVM` and `testTestsJVM` so it seemed like there was an intention to compile the tests as well. I'm fine with not compiling the tests, we should probably just be consistent.